### PR TITLE
Add continuation links to soar.md

### PR DESCRIPTION
**Pull request by TMickler**

### CODE REVIEW

**Comment by TMickler** [comment]
Please review this, @casey-collab

File: english/soar/soar.md [hunk 1]
```diff
@@ -3,4 +3,8 @@ No, not like a bird, like a jet!
 
 With the ground moving rapidly past you.
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Sing?](sing.md)
```
Comment (casey-collab) [inline english/soar/soar.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/soar/soar.md [hunk 2]
```diff
@@ -3,4 +3,8 @@ No, not like a bird, like a jet!
...
+[Check out Facebook?](http://www.facebook.com)
+[Love?](love/love.md)
```
Comment (casey-collab) [inline english/soar/soar.md]: This link is also invalid.

File: english/soar/soar.md [hunk 3]
```diff
@@ -3,4 +3,8 @@ No, not like a bird, like a jet!
 
 With the ground moving rapidly past you.
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Sing?](sing.md)
+[Check out Facebook?](http://www.facebook.com)
```
Comment (casey-collab) [inline english/soar/soar.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/soar/soar.md (hunk 2 above)
Comment (casey-collab) [inline english/soar/soar.md]: This link is also on the line directly after another link.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by TMickler** [comment]
Please review this, @casey-collab

File: english/soar/soar.md [hunk 4]
```diff
@@ -3,4 +3,8 @@ No, not like a bird, like a jet!
 
 With the ground moving rapidly past you.
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Sing?](english/sing/sing.md)
```
Comment (casey-collab) [inline english/soar/soar.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/soar/soar.md [hunk 5]
```diff
@@ -3,4 +3,8 @@ No, not like a bird, like a jet!
...
+[Check out Facebook?](https://www.facebook.com)
+[Love?](english/love/love.md)
```
Comment (casey-collab) [inline english/soar/soar.md]: This link is also invalid.

File: english/soar/soar.md [hunk 6]
```diff
@@ -3,4 +3,8 @@ No, not like a bird, like a jet!
 
 With the ground moving rapidly past you.
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Sing?](english/sing/sing.md)
+[Check out Facebook?](https://www.facebook.com)
```
Comment (casey-collab) [inline english/soar/soar.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/soar/soar.md (hunk 5 above)
Comment (casey-collab) [inline english/soar/soar.md]: This link is also on the line directly after another link.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**